features/erase.rst

```diff
@@ -0,0 +1,55 @@
+.. _erase:
+
+$erase resources
+===================
+
+Description
+-----------
+When Firely Server receives a DELETE request for a resource, it marks it as deleted in the database which makes it hidden from search results. However, the data is still present in the database. This approach is known as *soft deletion*. This comes in handy in scenarios when you want to recover accidentally deleted data. However, there are also scenarios when you *actually* want the data to be erased from the database. For that purpose, Firely Server provides the $erase operation.
+
+The $erase operation can be executed on a resource instance level and a resource version level.
+
+Examples
+^^^^^^^^
+
+Use the following request to erase all versions (including the historical versions) of the ``Patient/example`` resource from the database.
+
+::
+
+  POST <base-url>/Patient/example/$erase
+
+Use the following request to erase the specified version and all the older versions of the ``Patient/example`` resource from the database.
```

Comment:
Could you add a note that „older“ is determined based on lastUpdated? And maybe also add a concrete example on how to erase on history level between a range?

Reply:
```suggestion
Use the following request to erase the specified version ``xyz`` and all the older versions (based on meta.lastUpdated) of the ``Patient/example`` resource from the database.
```